README.md

```diff
@@ -1 +1,56 @@
-NL-Design-System
+# NL-Design-System
+
+The NL Design Skeleton application is designed for rapid application testing and prototype development on the NL Design System. It provides a basic skeleton application with full NL Design system functionality that any developer can easily extend, view locally and deploy to an online environment for demonstration purposes. The main benefits are:
```

Comment:
Misschien goed om hier _NL Design System_ een link te maken naar een plek waar hier iets meer informatie over staat? Denk voor de gemiddelde developer niet direct duidelijk wat hiermee mee wordt bedoeld.

Deze repository provide eigenlijk niet een skeleton application, het provide herbruikbare componenten die conform zijn aan de regels en richtlijnen van _NL Design System_. Het implementeren van deze componenten zorgt ervoor dat je gemakkelijk een applicatie kan maken die daardoor dus ook conform is aan deze richtlijnen. Dit betekent echter niet dat het een skeleton application an sich is (daarvoor hebben we de skeleton app repository).

Het _view locally_ _deploy_ en _demonstration_ gedeelte klopt daardoor ook niet helemaal. Alles wat deze repository doet (of deze npm package) is componenten aanbieden. Daar zit dus niets van deployment o.i.d. bij. De voornaamste voordelen zijn dus denk ik als volgt:

(1) Gemakkelijk en snel applicaties bouwen met pre-built componenten en
(2) Zeker zijn dat je voldoet aan de NL Design System regelgeving (doordat die regelgeving is verwerkt in deze componenten)